src/fileio/fileio_party.cpp

```diff
@@ -382,60 +382,62 @@ bool load_party_v2(fs::path file_to_load, cUniverse& real_univ, bool manual){
 			// Load scenario data
 			std::istream& fin = partyIn.getFile("save/scenario.txt");
 			if(!fin) {
-				if(manual) showError("Loading Blades of Exile save file failed.");
+				if(!preview) showError("Loading Blades of Exile save file failed.");
 				return false;
 			}
 			file.readFrom(fin);
 			univ.scenario.readFrom(file);
 		}
 		
-		{ // Then the "setup" array
-			std::istream& fin = partyIn.getFile("save/setup.dat");
-			if(!fin) {
-				if(manual) showError("Loading Blades of Exile save file failed.");
-				return false;
-			}
-			file.readFrom(fin);
-			auto& page = file[0];
-			for(size_t i = 0; i < univ.party.setup.size(); i++) {
-				page[std::to_string(i)].extract(univ.party.setup[i]);
+		if(!preview){
+			{ // Then the "setup" array
+				std::istream& fin = partyIn.getFile("save/setup.dat");
+				if(!fin) {
+					if(!preview) showError("Loading Blades of Exile save file failed.");
+					return false;
+				}
+				file.readFrom(fin);
+				auto& page = file[0];
+				for(size_t i = 0; i < univ.party.setup.size(); i++) {
+					page[std::to_string(i)].extract(univ.party.setup[i]);
+				}
 			}
-		}
 		
-		if(partyIn.hasFile("save/town.txt")) {
-			// Load town data
-			std::istream& fin = partyIn.getFile("save/town.txt");
-			if(!fin) {
-				if(manual) showError("Loading Blades of Exile save file failed.");
-				return false;
-			}
-			file.readFrom(fin);
-			univ.town.readFrom(file);
-		} else univ.party.town_num = 200;
+			if(partyIn.hasFile("save/town.txt")) {
+				// Load town data
+				std::istream& fin = partyIn.getFile("save/town.txt");
+				if(!fin) {
+					if(!preview) showError("Loading Blades of Exile save file failed.");
+					return false;
+				}
+				file.readFrom(fin);
+				univ.town.readFrom(file);
+			} else univ.party.town_num = 200;
 
-		if (partyIn.hasFile("save/townmaps.dat")) {
-			// Read town maps
-			std::istream& fin2 = partyIn.getFile("save/townmaps.dat");
-			for(int i = 0; i < univ.scenario.towns.size(); i++) {
-				for(int j = 0; j < univ.scenario.towns[i]->max_dim; j++) {
-					fin2 >> univ.scenario.towns[i]->maps[j];
+			if (partyIn.hasFile("save/townmaps.dat")) {
+				// Read town maps
+				std::istream& fin2 = partyIn.getFile("save/townmaps.dat");
+				for(int i = 0; i < univ.scenario.towns.size(); i++) {
+					for(int j = 0; j < univ.scenario.towns[i]->max_dim; j++) {
+						fin2 >> univ.scenario.towns[i]->maps[j];
+					}
 				}
 			}
-		}
-		// Load outdoors data
-		std::istream& fin = partyIn.getFile("save/out.txt");
-		if(!fin) {
-			if(manual) showError("Loading Blades of Exile save file failed.");
-			return false;
-		}
-		univ.out.readFrom(fin);
+			// Load outdoors data
+			std::istream& fin = partyIn.getFile("save/out.txt");
+			if(!fin) {
+				if(!preview) showError("Loading Blades of Exile save file failed.");
```

Comment:
It looks like this check is now inside a larger `if(preview)` block, making it redundant.